test/lint/lint-includes.sh

```diff
@@ -68,7 +68,6 @@ EXPECTED_BOOST_INCLUDES=(
     boost/test/unit_test.hpp
     boost/thread.hpp
     boost/thread/condition_variable.hpp
```

Comment:
According to PR title `boost/thread/condition_variable.hpp` should go as well? :-)

Reply:
I only remove them in checkqueue, the another place to use it is in scheduler.